docusaurus/website/docusaurus.config.ts

```diff
@@ -216,6 +216,12 @@ const config: Config = {
       version: gitHash,
       environment: process.env.DEV_PORTAL_FARO_CONFIG_ENVIRONMENT,
     },
+    oneTrust: {
+      enabled: true,
+      scriptSrc: 'https://cdn.cookielaw.org/scripttemplates/otSDKStub.js',
+      domainId: '019644f3-5dcf-741c-8b6d-42fb8feae57f',
```

Comment:
I think you should put `domainId` into [dev](https://github.com/grafana/plugin-tools/blob/main/docusaurus/website/.env.development) and [prod](https://github.com/grafana/plugin-tools/blob/main/docusaurus/website/.env.production) files

so that you can use a different id for dev domain